src/data-dictionary/events/NrAiIssue/sources.md

```diff
@@ -0,0 +1,8 @@
+---
+name: sources
+type: attribute
+events:
+  - NrAiIssue
+---
+
+If applicable, the source of the issue.
```

Comment:
i think as a general point, we can avoid/erase 'if applicable' or 'present when...' types of notes. Reason being: many of our attributes are only present when x/y/z things are true, so it can go without saying that they may or may not be present. This is just a general tip and I think we talk about this and other aspects in the data dictionary style guide.

Reply:
ok, there are several "If applicable"